plugin/pkg/admission/storage/persistentvolume/label/admission_test.go

```diff
@@ -173,9 +173,9 @@ func Test_PVLAdmission(t *testing.T) {
 			name:    "AWS EBS PV labeled correctly",
 			handler: newPersistentVolumeLabel(),
 			pvlabeler: mockVolumeLabels(map[string]string{
-				"a":                       "1",
-				"b":                       "2",
-				v1.LabelZoneFailureDomain: "1__2__3",
+				"a":                                 "1",
+				"b":                                 "2",
+				v1.DeprecatedLabelZoneFailureDomain: "1__2__3",
```

Comment:
we either should:

- test using the deprecated label and GA one
- test using the GA one